features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/extensions/store/app/am_store/app.js

```diff
@@ -24,6 +24,10 @@ app.server = function(ctx) {
                 url: 'widget',
                 path: 'widget.jag',
                 secured:false
+            }, {
+                url: 'applications',
+                path: 'applications.jag',
+                secured:false
```

Comment:
what will be the impact of setting secured attribute to false. Will it make users to access this page without authenticating